.npmignore

```diff
@@ -12,6 +12,7 @@
 !index.js
 !esbuild.js
 !init.js
+!initialize.mjs
```

Comment:
Since this is a loader, and the fact that it also adds initialization is an implementation detail, wouldn't it be best to call this `load.mjs` or something similar instead?

Reply:
When used with `--import`, it's not a loader, but registers one alongside doing the usual `/init` stuff. I'd rather not call it `load` so as to not make that conflation, even if it can be used that way.